src/main/java/lotto/MatchedNumbers.java

```diff
@@ -0,0 +1,24 @@
+package lotto;
+
+public enum MatchedNumbers {
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000);
```

Comment:
```suggestion
    SIX(6, 2_000_000_000);
```
자바에서는 큰 단위에 대해서 콤마를 사용하지는 못하지만 이렇게 언더바(`_`)를 이용해서 숫자의 가독성을 높혀줄 수 있습니다.